ModList/UI/ViewControllers/ModListViewController.cs

```diff
@@ -93,23 +93,48 @@ protected override void DidDeactivate(bool removedFromHierarchy, bool screenSyst
 
         internal async void OnAnimationFinish()
         {
-            if (CustomListTableData != null && CustomListTableData.data?.Count != modProviderService.PluginList.Count)
+            if (ListValues.Count != modProviderService.PluginList.Count)
             {
                 await SiraUtil.Extras.Utilities.PauseChamp;
                 ReloadViewList();
+                Loaded = true;
             }
-            Loaded = true;
         }
 
         internal void ReloadViewList()
         {
-            ListValues.Clear();
-            ListValues.AddRange(modProviderService.PluginList.Select(p => new PluginCellViewController(p)));
+            ListValues = new List<PluginInformation>(modProviderService.PluginList);
```

Comment:
Why re-create the list when clear/addrange could reuse the underlying allocations?

Reply:
Oh if they're not equivalent I will change it back